lib/libzpool/uzfs_mgmt.c

```diff
@@ -385,6 +386,101 @@ uzfs_create_dataset(spa_t *spa, char *ds_name, uint64_t vol_size,
 	return (error);
 }
 
+/* uZFS Zvol create call back function */
+int
+uzfs_zvol_create_cb(const char *ds_name, void *arg)
```

Comment:
this seems to be duplicated code, can be merged with other function.

Reply:
True. The other API which we are talking about has been developed very recently and it will take time for me to soak those change.  Since this code has wider impact on export/create/destroy i.e we have to take care of refcount increments and decrements on dataset as part of create and destroy respectively and since  this code is already tested and very much stable, I am little apprehensive  to merge both of them right away. Will open an issue and fix it. I hope that is fine.